Gemfile

```diff
@@ -33,6 +33,7 @@ group :development, :test do
   gem 'rspec-rails'
   gem 'faker'
   gem 'factory_bot_rails'
+  gem 'database_cleaner'
```

Comment:
why database cleaner?

Reply:
I'm using database cleaner to clean the database after every test